README.md

```diff
@@ -1,5 +1,16 @@
+# Corona games
+
+This is a collection of games that may help you and your friends kill the boredom of COVID19 quarantine.
+
+
+## Development 
+Install dependencies `npm install`.
+
 Run react dev server with `npm start`.
 
-Run game server with: `node -r esm src/server.js`
+Run game server with: `node -r esm src/server.js`.
+
+Configure player id with query param `id=some_player_id` (default is `0`).
```

Comment:
I'm pretty sure it doesn't work anymore 👀

Reply:
But it's possible to show debug overlay with `debug=true` query param

Reply:
Thanks! Added ;)